addons/message_survey/OpenSurveyURL.js

```diff
@@ -0,0 +1,10 @@
+((vpn) => {
+  // TODO once OS is available to addons
+  const desktop = true;
```

Comment:
To be tested:
`const desktop = [ "linux", "macos", "windows"].contains(api.env.platform);`